storage/imx6ull-flash/imx6ull-flash-test.c

```diff
@@ -497,8 +498,8 @@ void test_meta(void)
 	const unsigned int blockno = 0;
 	uint32_t paddr = blockno * 64;
 
-	data = mmap(NULL, _PAGE_SIZE * 2, PROT_READ | PROT_WRITE, MAP_UNCACHED, OID_CONTIGUOUS, 0);
-	meta = data + _PAGE_SIZE;
+	data = mmap(NULL, pagemapsz, PROT_READ | PROT_WRITE, MAP_UNCACHED, OID_CONTIGUOUS, 0);
+	meta = data + flashinfo.writesz;
```

Comment:
not sure if we shouldn't have `_PAGE_SIZE` here as DMA buffers need to be (RAM) page-aligned regardless of `flashinfo.writesz`. There was a silent assumption here that `flashinfo.writesz <= _PAGE_SIZE` (we can probably add assert / check it when starting tests just to be sure).

This affects other places where `mmap`'ing data + meta together.

Reply:
I agree, `meta` should be explicitly initialized to page aligned address. I'm going to add the proposed assert in `main()` and revert this change.